.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Unit Tests
+
+on:
+  push:
+  pull_request:
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+# Cancels all previous workflow runs for the same branch that have not yet completed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: ['7.3', '7.4', '8.0', '8.1']
+        composer-flags: ['']
+
+        include:
+          - php: '8.2'
+            composer-flags: '--ignore-platform-reqs'
+
+    name: PHP ${{ matrix.php }}
+
+    continue-on-error: ${{ matrix.php == '8.2' }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.3.4
```

Comment:
Please use just `@v2` here, to reduce the maintenance burden.

Reply:
Done, sorry, copy/pasta. I normally use `v2` myself as well.